tools/github-issues/Azure.Sdk.Tools.GitHubIssues/Helpers/GitHubHelpers.cs

```diff
@@ -1,5 +1,8 @@
 ﻿using Azure.Sdk.Tools.GitHubIssues.Services.Configuration;
+using ComposableAsync;
```

Comment:
What does this do?

Reply:
That's just the namespace in RateLimiter (I think its just a case of Namespace != Package name).

Reply:
Oh ... I see, interesting. Why are there two namespaces (RateLimiter and ComposableAsync). Will check that out.

Reply:
Looks like ```GetAwaiter()``` is defined in ```ComposableAsync.Core```.